superset/config.py

```diff
@@ -841,7 +841,7 @@ class D3Format(TypedDict, total=False):
 # Console Log Settings
 
 LOG_FORMAT = "%(asctime)s:%(levelname)s:%(name)s:%(message)s"
-LOG_LEVEL = "DEBUG"
+LOG_LEVEL = "INFO"
```

Comment:
Would you mind adding a line item to `UPDATING.md`?

Reply:
Done in https://github.com/apache/superset/pull/27777/commits/85ca2cd9f8d5148eb90232e4672b939b515c45fb